website/docs/feature-flag-tutorials/python/examples.md

```diff
@@ -0,0 +1,508 @@
+
+We built multiple features into Unleash, an open-source feature flag platform, to address the complexities of releasing code and managing feature flags along the way. This tutorial will explore the following:
+
+1. [Gradual rollouts](#gradual-rollouts-for-python-apps)
+2. [Canary deployments](#canary-deployments-in-python)
+3. [Server-side A/B testing](#server-side-ab-testing-in-python)
+4. [Feature flag metrics & reporting](#feature-flag-analytics-and-reporting-in-python)
+5. [Feature flag audit logs](#feature-flag-audit-logs-in-python)
+6. [Flag automation & workflow integration](#flag-automation--workflow-integration-for-python-apps)
+7. [Common usage examples of Python feature flags](#common-usage-examples-of-python-feature-flags)
+
+
+## Gradual Rollouts for Python Apps
+
+It is common to use feature flags to roll out changes to a percentage of users, and we can use Unleash to do a gradual rollout with a Python-based app. 
+
+In our Python tutorial, the flag controls the release of a feature allowing users to delete surveys they create. To further customize that, you can modify the basic setup to adjust the percentage of users who experience this feature with a gradual rollout. The users split between the delete button being visible or not is cached so their user experience will remain consistent. 
+
+Navigate to the gradual rollout page where you can edit your strategy.
+
+![The "edit strategy" button uses a pencil icon and is located on every strategy.](/img/react-ex-grad-rollout-edit.png)
+
+Adjust the percentage of users to 50% or whichever value you choose, and refresh your app in the browser to see if your user has the new feature experience. This might take 30 seconds for a server-side app to propagate.
+
+![A gradual rollout form can allow you to customize your flag strategy.](/img/react-ex-grad-rollout-form.png)
+
+
+You can achieve this same result using our API with the following code:
+
+```py
+import requests
+import json
+
+url = "<your-unleash-url>/api/admin/projects/:projectId/features/:featureName/environments/:environment/strategies"
+
+payload = json.dumps({
+  "name": "flexibleRollout",
+  "disabled": False,
+  "constraints": [],
+  "variants": [],
+  "parameters": {
+    "groupId": "delete_survey_flag",
+    "rollout": "50",
+    "stickiness": "sessionId"
+  },
+  "segments": []
+})
+
+headers = {
+  'Content-Type': 'application/json',
+  'Accept': 'application/json',
+  'Authorization': '<API_KEY_VALUE>'
+}
+
+response = requests.request("POST", url, headers=headers, data=payload)
+
+print(response.text)
+```
+
+Learn more about [gradual rollouts in our docs](/reference/activation-strategies). Also, learn more about our [API for creating a new strategy](/reference/api/unleash/add-feature-strategy) for your flag.
+
```

Comment:
Technically it's not the same thing, since what you've shown in the UI is editing the existing strategy, and this API request adds a new strategy. 

To confirm, you'll notice on the side panel when editing a strategy that it mentions the `PUT` method and not `POST`: 
![image](https://github.com/Unleash/unleash/assets/14320932/709df3cc-9c52-41fc-9912-77479c952857)

I think the endpoint you're looking for is this one: https://unleash-docs-git-python-examples-unleash-team.vercel.app/reference/api/unleash/update-feature-strategy